test/extended/networking/livemigration.go

```diff
@@ -188,13 +199,14 @@ var _ = Describe("[sig-network][OCPFeatureGate:PersistentIPsForVirtualization][F
 					nad := generateNAD(netConfig)
 					By(fmt.Sprintf("Creating NetworkAttachmentDefinitions %s/%s", nad.Namespace, nad.Name))
 					_, err := nadClient.NetworkAttachmentDefinitions(c.namespace).Create(context.Background(), nad, metav1.CreateOptions{})
-					Expect(err).NotTo((HaveOccurred()))
+					Expect(err).NotTo(HaveOccurred())
 				}),
-				Entry("UserDefinedNetwork", func(c networkAttachmentConfigParams) {
+				Entry("UserDefinedNetwork", func(c networkAttachmentConfigParams) networkAttachmentConfig {
 					udnManifest := generateUserDefinedNetworkManifest(&c)
 					By(fmt.Sprintf("Creating UserDefinedNetwork %s/%s", c.namespace, c.name))
 					Expect(applyManifest(c.namespace, udnManifest)).To(Succeed())
 					Expect(waitForUserDefinedNetworkReady(c.namespace, c.name, udnCrReadyTimeout)).To(Succeed())
+					return networkAttachmentConfig{networkAttachmentConfigParams{networkName: c.namespace + "." + c.name}}
```

Comment:
@ormergi how can I have access to the name of the network ? 

Right now, the only thing I think I can do is to read the corresponding NAD, unmarshal, and return that ... 

Is there something simpler ?

Reply:
@npinaeva how about you ?

Reply:
Fetched it from the NAD generated for the UDN.

I think we can assume the name is the same (i.e. NAD.metadata.Name == UDN.metadata.Name) 

If that changes, we'll need to be able to derive it somehow. Not thinking much about that, because the change must be backwards compatible. 

@npinaeva

Reply:
I would fetch the NAD that created by the UDN controller and parse the network-name it got.
Doing so ensure tests wont break even if network-name will be changed, no need to go back to tests and do some adjustments.

Reply:
Right, that would make sense. I have the owner reference for that, right ? @ormergi

Reply:
Yes, you can also fetch the NAD with the meta.name as the UDN object as well